src/SwagVoucherFunding.php

```diff
@@ -39,25 +39,73 @@ private function createConfiguration(Context $context): void
             [
                 'id' => Uuid::randomHex(),
                 'configurationKey' => $this->getName() . '.config.pdfTemplate',
-                'configurationValue' => $this->pdfTemplate()
+                'configurationValue' => $this->getPdfTemplate()
             ]
         ], $context);
 
         $systemConfigRepository->create([
             [
                 'id' => Uuid::randomHex(),
-                'configurationKey' => $this->getName() . '.config.subject',
+                'configurationKey' => $this->getName() . '.config.customerSubject',
                 'configurationValue' => 'Your voucher with {{ merchant.publicCompanyName }}'
             ]
         ], $context);
 
         $systemConfigRepository->create([
             [
                 'id' => Uuid::randomHex(),
-                'configurationKey' => $this->getName() . '.config.senderName',
+                'configurationKey' => $this->getName() . '.config.customerSenderName',
                 'configurationValue' => '{{ merchant.publicCompanyName }}'
             ]
         ], $context);
+
+        $systemConfigRepository->create([
+            [
+                'id' => Uuid::randomHex(),
+                'configurationKey' => $this->getName() . '.config.customerHtmlTemplate',
+                'configurationValue' => $this->getCustomerMailHTMLTemplate()
+            ]
+        ], $context);
+
+        $systemConfigRepository->create([
+            [
+                'id' => Uuid::randomHex(),
+                'configurationKey' => $this->getName() . '.config.customerPlainTemplate',
+                'configurationValue' => $this->getCustomerMailPlainTemplate()
+            ]
+        ], $context);
+
+        $systemConfigRepository->create([
+            [
+                'id' => Uuid::randomHex(),
+                'configurationKey' => $this->getName() . '.config.merchantSubject',
+                'configurationValue' => 'We have just issued a voucher for your customer'
```

Comment:
Btw just FYI: you can define default values in the config.xml directly using `<value>text</value>` 😊